docs/user-guide/cli-updatingcli.md

```diff
@@ -1,98 +1,149 @@
+   You have successfully updated to the Zowe CLI V3-LTS version.
 
-## Identify the currently installed version of Zowe CLI
+5. See [Next steps](#next-steps) for recommended tasks after installation.
 
-Issue the following command (case-sensitive):
+### Updating from a local package
 
-```
-zowe -V
-```
+To update Zowe CLI core and Zowe CLI plug-ins from an offline (`.tgz`), local package, uninstall your current package then reinstall from a new package.
 
-## Identify the currently installed versions of Zowe CLI plug-ins
+1. To uninstall Zowe CLI and Zowe CLI plug-ins, follow the instructions in [Uninstalling Zowe CLI and Zowe CLI Plug-ins](../user-guide/cli-uninstall.md).
 
-Issue the following command:
+2. To install a new package for Zowe CLI and Zowe CLI plug-ins, follow the instructions in [Installing Zowe CLI and Zowe CLI plug-ins from a local package](../user-guide/cli-installcli.md#installing-zowe-cli-and-zowe-cli-plug-ins-from-a-local-package).
 
-```
-zowe plugins list
-```
+3. See [Next steps](#next-steps) for recommended tasks after installation.
+
+## Updating or reverting Zowe CLI and Zowe CLI plug-ins to a specific version
+
+If necessary, you can update or revert Zowe CLI to a known, previously released version.
 
-## Update Zowe CLI from the online registry
+### Updating or reverting from an npm online registry
 
-You can update Zowe CLI to the latest version from the online registry on Windows, Mac, and Linux computers.
+1. To update or revert Zowe CLI and Zowe CLI plug-ins to a specific known version, open a command line window and issue the following command:
 
-**Note:** The following steps assume that you previously installed the CLI as described in [Installing Zowe CLI from an online registry](cli-installcli.md#installing-zowe-cli-from-an-online-registry).
+   ```
+   npm install -g @zowe/cli@<X.Y.Z>
+   ```
+
+      - `<X.Y.Z>`
+
+      Specifies the version that Zowe CLI or Zowe CLI plug-ins are to be updated or reverted to. For example, `npm install -g @zowe/cli@8.0.0`.
+
+2. See [Next steps](#next-steps) for recommended tasks after installation.
+
+### Updating or reverting from a local package
+
+1. Uninstall Zowe CLI and Zowe CLI plug-ins. Follow the instructions in [Uninstalling Zowe CLI and Zowe CLI Plug-ins](../user-guide/cli-uninstall.md).
+
+2. Navigate to [Download Zowe](https://www.zowe.org/download.html). Select the specific release you want to update or revert to from **All Zowe V2.x Releases** or **All Zowe V3.x Releases**.
+
+   The End User License Agreement for Zowe displays.
+
+3. Read the End User License Agreement for Zowe and click **I agree** to download the core package.
 
-1. Update Zowe CLI. Open a command line window and issue the following command:
+    `zowe-cli-package-next-YYYYMMDD.zip` is downloaded to your computer (where YYYYMMDD indicates the year, month, and day of the build).
```

Comment:
the package will be named `zowe-cli-package-x.yy.z.zip` (where x is the major version, yy is the minor version, and z is the patch version corresponding to a given Zowe release)